src/app/search/state/search.service.ts

```diff
@@ -555,7 +555,7 @@ export class SearchService {
         'descriptor_type_versions.keyword',
         'Indicates that the tool or workflow contains at least one version that is written with the workflow language version',
       ],
-      ['openData', 'Indicates whether an entry has a test parameter file with all publicly accessible data.'],
+      ['openData', 'Indicates whether an entry can be run with a provided test parameter file and no additional access permissions.'],
```

Comment:
Technically, this is "can be run with (a provided test parameter file with no additional access permissions) or (no additional access permissions" which is clearly terrible. 
How about "can be run with no additional access permissions, potentially with a test parameter file referencing open data"?